File: app/src/main/java/com/ruanmeng/myapplication/MyApplication.java
package com.ruanmeng.myapplication;

import android.app.Application;

import com.ruanmeng.common.Constant;

/**
 * Created by Andriod0729 on 2018/12/15.
 */

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        // 注册Context,在Application中
        Constant.getInstance().init(this);
    }
}

File: app/src/main/java/com/mydeerlet/androidutils/MyApplication.kt
package com.mydeerlet.androidutils

import android.app.Application

class MyApplication :Application() {

    override fun onCreate() {
        super.onCreate()

//        if (Constant.DEBUG_MODE) {           // These two lines must be written before init, otherwise these configurations will be invalid in the init process
//            ARouter.openLog()     // Print log
//            ARouter.openDebug()   // Turn on debugging mode (If you are running in InstantRun mode, you must turn on debug mode! Online version needs to be closed, otherwise there is a security risk)
//        }
//        ARouter.init(this) // As early as possible, it is recommended to initialize in the Application


//        ApiService.init("http://123.56.232.18:8080/serverdemo",null)
    }
}